pkgs/development/libraries/kde-frameworks/kpackage/allow-external-paths.patch

```diff
@@ -1,11 +1,11 @@
-Index: kpackage-5.30.0/src/kpackage/package.cpp
-===================================================================
---- kpackage-5.30.0.orig/src/kpackage/package.cpp
-+++ kpackage-5.30.0/src/kpackage/package.cpp
+diff --git a/src/kpackage/package.cpp b/src/kpackage/package.cpp
+index 5aec9fd..b15c933 100644
+--- a/src/kpackage/package.cpp
++++ b/src/kpackage/package.cpp
 @@ -820,7 +820,7 @@ PackagePrivate::PackagePrivate()
      : QSharedData(),
-       fallbackPackage(0),
-       metadata(0),
+       fallbackPackage(nullptr),
+       metadata(nullptr),
 -      externalPaths(false),
 +      externalPaths(true),
```

Comment:
I see now. The coloring is also applied to unchanged files. That confused me.
I've no problems with the patch as I read it, but I've not tested it.